webapp-django/crashstats/crashstats/tests/test_jinja_helpers.py

```diff
@@ -86,29 +89,90 @@ def test_show_bug_link_with_cache(self):
 
 class TestBugzillaSubmitURL(TestCase):
 
-    def test_basic_url(self):
-        url = bugzilla_submit_url()
-        eq_(
-            url,
-            'https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__'
-        )
+    @staticmethod
+    def _create_report(**overrides):
+        default = {
+            # 'os_name': 'Windows',
+            # 'os_pretty_version': '',
+            'signature': '$&#;deadbeef',
+            'uuid': '00000000-0000-0000-0000-000000000000',
+            'cpu_name': 'x86'
+        }
+        return dict(default, **overrides)
+
+    @staticmethod
+    def _extract_query_string(url):
+        return cgi.parse_qs(urlparse.urlparse(url).query)
```

Comment:
This is deprecated, use ``urlparse.parse_qs`` instead. See https://docs.python.org/2/library/cgi.html#cgi.parse_qs